.github/workflows/build.yml

```
@@ -166,135 +148,87 @@ jobs:
+          docker compose -f docker-compose-deploy/docker-compose.yml logs
+          docker compose -f docker-compose-deploy/docker-compose.yml down
+          exit 1
+
+      - name: Build and push Controller multi-arch image
+        if: github.event_name == 'push'
+        uses: docker/build-push-action@v6
+        with:
+          context: .
+          file: ./Dockerfile.controller
+          push: true
+          tags: ${{ steps.meta_controller.outputs.tags }}
```

actionlint (1.7.7)

property "meta_controller" is not defined in object type {meta: {conclusion: string; outcome: string; outputs: {annotations: string; bake-file: string; bake-file-annotations: string; bake-file-labels: string; bake-file-tags: string; json: string; labels: string; tags: string; version: string}}}

(expression)